doc/learning/commitmessage/index.rst

```diff
@@ -0,0 +1,111 @@
+Commit Message Format
+=====================
+
+We have very precise rules over how our Git commit messages must be formatted.
+This format leads to **easier to read commit history**.
+Each commit message consists of a **header**, a **body**, and a **footer**.
+
+.. code-block:: text
+
+    <header>
+    <BLANK LINE>
+    <body>
+    <BLANK LINE>
+    <footer>
+
+Commit Message Header
+---------------------
+
+.. code-block:: text
+
+    <type>: <subject line>
```

Comment:
We probably don't want any additional metadata like <type> in the subject line. So this particular check would need to be reversed.